lore/io/connection.py

```diff
@@ -134,7 +134,6 @@ def comment_sql_calls(conn, cursor, statement, parameters, context, executemany)
         @event.listens_for(self._engine, "after_cursor_execute")
         def time_sql_calls(conn, cursor, statement, parameters, context, executemany):
             total = datetime.now() - conn.info['query_start_time'].pop(-1)
-            logger.info("SQL: %s" % total)
```

Comment:
Why are we still using `lore`? It's been deprecated for a while now. cc @rajpalparyani

Reply:
This service has a foot in the grave and we're trying to replace it.